01-TodoList/00-Design/code-review/todolist nest 버전 코드 리뷰 - 2.md

```
@@ -0,0 +1,882 @@
++
++        return new BadRequestException({
++          success: false,
++          error: {
++            code: 'VALIDATION_ERROR',
++            message: messages[0]?.errors[0] || 'Validation failed',
++            details: messages,
++          },
++        });
+```
+
+_⚠️ Potential issue_ | _🟡 Minor_
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis style
Expected: asterisk; Actual: underscore

(MD049, emphasis-style)